openquake/hazardlib/calc/disagg.py

```diff
@@ -337,7 +337,7 @@ def disaggregation(
              'filter_distance': filter_distance, 'imtls': {str(imt): [iml]}})
         contexts.RuptureContext.temporal_occurrence_model = (
             srcs[0].temporal_occurrence_model)
-        rdata = contexts.RupData(cmaker).from_srcs(srcs, sitecol)
+        rdata = contexts.RupData(cmaker, **kwargs).from_srcs(srcs, sitecol)
```

Comment:
This looks wrong to me, RupData does not accept kwargs. Strange that no test broke.